snappi_ixnetwork/trafficitem.py

```diff
@@ -1,4 +1,4 @@
-import time
+import json
 from snappi_ixnetwork.timer import Timer
 from snappi_ixnetwork.customfield import CustomField
```

Comment:
We need to modify customfield.py

Reply:
Yes, right now custom field is not used, while implementing ethernet pause and ipv4 priority, will modify